package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>fp</name>
-  <version>0.0.1</version>
+  <version>0.1.0</version>
```

Comment:
Why is there still a packag.xml file if this PR is supposed to remove all the ROS stuff?

Reply:
This is a good point, I will try removing it.